.github/workflows/python-release.yaml

```diff
@@ -113,13 +111,17 @@ jobs:
             }
 
   macos:
-    runs-on: macos-12-xl
+    runs-on: ${{ matrix.config.runner }}
     strategy:
       fail-fast: true
       matrix:
-        target: [x86_64, aarch64]
+        config:
+        - target: x86_64-apple-darwin
+          runner: macos-13-large
+        - target: aarch64-apple-darwin
+          runner: macos-13-xlarge
 
-    name: Build Python Bindings (macos ${{ matrix.target }})
+    name: Build Python Bindings (macos ${{ matrix.config.target }})
```

Comment:
I think we need to make this change again below (line 154 if I'm reading the diff right) where we post to slack